dhctl/pkg/kubernetes/actions/deckhouse/log.go

```diff
@@ -268,7 +269,13 @@ func (d *LogPrinter) Print(ctx context.Context) (bool, error) {
 		return false, err
 	}
 
-	logOptions := corev1.PodLogOptions{Container: "deckhouse", TailLines: int64Pointer(5)}
+	logOptions := corev1.PodLogOptions{
+		Container: "deckhouse",
+		TailLines: int64Pointer(5),
+
+		InsecureSkipTLSVerifyBackend: true,
```

Comment:
Let's leave a comment on why we need this option enabled. It is not evident at first glance.